sdk_lib/sdk_entry.sh

```diff
@@ -10,9 +10,29 @@ fi
 
 chown -R sdk:sdk /home/sdk
 
-# Fix up SDK repo configuration to use the new coreos-overlay name.
-sed -i -r 's/^\[coreos\]/[coreos-overlay]/' /etc/portage/repos.conf/coreos.conf 2>/dev/null
-sed -i -r '/^masters =/s/\bcoreos(\s|$)/coreos-overlay\1/g' /usr/local/portage/crossdev/metadata/layout.conf 2>/dev/null
+# Fix up SDK repo configuration to use the new layout.
```

Comment:
This entry block has now gotten scary.

We used to rely on `update_chroot --setuponly` for setting up the sdk, how about calling it from here. It has the same definitions.

Reply:
...except calling update_chroot breaks the version check that is then used to decide whether to update the board or not. :confused: Any ideas?

Reply:
I guess I could move that update_chroot call inside that block because we'd also only want to run that when:

```
if [ "${version}" != "${DISTRIB_RELEASE}" ] ; then
```